### PR TITLE
Documenting datasets and fixing LaTeX issues

### PR DESCRIPTION
**Pull request by gvegayon**
Hey @lailylajiang, so I'm fixing the issues here. The PR is still in progress, I still see some errors and issues when running R CMD check on RStudio. Will continue later. A couple of outstanding issues I cannot address here:

1. There are a couple of datasets that you used in a previous version of the package which are no longer included (see the errors in the check).
2. A couple of arguments still need to be defined, I've added them as `TBD`.
3. The issue with the undefined `value` variable has to do with a `mutate()` call you make. Using `dplyr` in an R package is not encouraged in general. If you want to use it, you need to make sure you follow their recommendations to avoid issues like that :).
4. Finally, while you are only listing about 4 dependencies, this package has about 100 indirect dependencies, which makes it vulnerable if any of those fails (i.e. CRAN could remove it). Make sure the packages you are using are 100% needed. Usually, if an R base function exists for that, or you are simply using a couple of functions of the package, then it is a good idea to use an alternative instead.

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -14,7 +14,9 @@
 #' @references Locke, Adam E., et al. Genetic studies of body mass index yield new insights for obesity biology. Nature 518.7538 (2015): 197-206.
 #' @references Xue, Angli, et al. Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. Nature communications 9.1 (2018): 1-14.
 #' @references Sudlow, Cathie, et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. Plos med 12.3 (2015): e1001779.
-"MI"
```
Comment (gvegayon) [inline R/data.R]: I explicitly declared the name of the data object and then added aliases to make sure R was able to recognize the objects as documented in this manual.

Comment (lailylajiang) [reply R/data.R]: Thanks George!

File: R/data.R [hunk 2]
```diff
@@ -4,8 +4,8 @@
 #'
 #' @format The \code{MI} object is a set of data sets which was used to estimate the causal effect of body mass index and type 2 diabetes on the risk of myocardial infarction.
 #' \describe{
-#'    \item{MI.marginal.Amatrix}{The marginal Â matrix. Column one and two are the marginal estimates of the SNPs on body mass index from GIANT consortium (n = 339,224) (Locke et al., 2015) and type 2 diabetes from DIAGRAM+GERA+UKB (n = 659,316) (Xue et al., 2018), respectively}
-#'    \item{MI.Amatrix}{The conditional Â matrix composed by JAM and the marginal Â matrix. Column one and two are the conditional effect estimates of the SNPs on body mass index and type 2 diabetes, respectively.}
```
Comment (gvegayon) [inline R/data.R]: @lailylajiang LaTeX symbols and text should be included using either \eqn{}, which is equivalent to $$, or \deqn{}, which is equivalent to $$$$. eqn and deqn have two arguments, the second one optional. The first argument is the latex code you want to include, and the second argument is the ascii equivalent, which is usually a verbatim way to represent the symbol. The latex text will be included in the pdf manual of the package, whereas the ascii is included in the help files.

File: R/data.R [hunk 3]
```diff
@@ -14,32 +14,37 @@
...
+#'     \item{GTEx.PrCa.Amatrix}{The conditional A matrix with 167 genes and 447 eQTLs, which was composed by SuSiE JAM and the raw data of \eqn{\hat{A}}{A hat}  matrix.}
 #'     \item{GTEx.PrCa.Geno}{The reference genotype data for the 447 eQTLs from the European-ancestry population in 1000 Genome Project (Consortium, 2015)}
 #'     \item{GTEx.PrCa.betas.gwas}{The b vector. The association estimates between eQTLs and the risk of prostate cancer from (Schumacher et al., 2018)}
 #'     \item{GTEx.PrCa.betas.se.gwas}{The se(b) vector from (Schumacher et al., 2018)}
-#'     \item{GTEx.PrCa.pvalue.gwas}{The pvalues vector of the association estimates between selected SNPs and the risk of prostate cancer from (Schumacher et al., 2018)}
+#'     \item{GTEx.PrCa.pvalues.gwas}{The pvalues vector of the association estimates between selected SNPs and the risk of prostate cancer from (Schumacher et al., 2018)}
 #'     \item{GTEx.PrCa.maf.gwas}{The vector of the effect allele frequency of the SNPs from (Schumacher et al., 2018)}
-#'     \item{GTEx.PrCa.rsid}{The RSID of the eQTLs}
+#'     \item{GTEx.PrCa.rsid}{NOT PRESENT The RSID of the eQTLs}
+#'     \item{GTEx.PrCa.marginal.selected}{TBD}
```
Comment (gvegayon) [inline R/data.R]: @lailylajiang  There are a few inconsistencies on the datasets included. I fixed a couple, but there seems to be some missing data (not sure). You need to check that out :).

File: R/data.R (hunk 2 above)
Comment (lailylajiang) [reply R/data.R]: Got it

**Review by lailylajiang — APPROVED** [review]

**Review by lailylajiang — APPROVED** [review]

**Comment by lailylajiang** [comment]
Hi @gvegayon, in terms of your last comment about the dependencies. I agree but am not sure how to remove the unnecessary ones. I don't quite know why the package imports them during the build-up. How could I remove the unnecessary ones? Thanks!

**Pull request merged**